pkg/operator/highcpuusagealertcontroller/cpuPartitioningAlertRenderer.go

```diff
@@ -0,0 +1,93 @@
+package highcpuusagealertcontroller
+
+import (
+	"bytes"
+	"context"
+	"strconv"
+	"time"
+
+	configv1informers "github.com/openshift/client-go/config/informers/externalversions/config/v1"
+	configlistersv1 "github.com/openshift/client-go/config/listers/config/v1"
+	"github.com/openshift/library-go/pkg/controller/factory"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceapply"
+	"github.com/openshift/library-go/pkg/operator/resource/resourceread"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic/dynamicinformer"
+	"k8s.io/client-go/tools/cache"
+
+	v1 "github.com/openshift/api/config/v1"
+	"github.com/openshift/cluster-kube-apiserver-operator/bindata"
+	"github.com/openshift/library-go/pkg/operator/events"
+	"k8s.io/client-go/dynamic"
+)
+
+type HighCPUUsageController struct {
+	client                   dynamic.Interface
+	infraLister              configlistersv1.InfrastructureLister
+	performanceProfileLister cache.GenericLister
+}
+
+func NewHighCPUUsageAlertController(infraInformer configv1informers.InfrastructureInformer, client dynamic.Interface, genericInformer dynamicinformer.DynamicSharedInformerFactory, recorder events.Recorder) factory.Controller {
+	performanceInformer := genericInformer.ForResource(performanceGroup)
```

Comment:
In the previous comment you said the the informer can be disabled by setting a capability.
Thus we should make this informer optional.
It should be not created when the capability was set.

Reply:
OK, so after our discussion this morning about the capability being enabled at runtime, and after taking more time to think it through and weighing some trade-offs, I think we could:

- ensure the alert is reconciled every 5 or 10 minutes.
- check whether the capability is enabled or disabled.
- if the capability is enabled, list data directly from the server (as you previously suggested).
- if the capability is disabled, use the default value.
- document the previous 3 steps
- reconcile the alert.

WDYT ?

Reply:
> Do I hear it right that are suggesting to use dynamic client instead of dynamic informer for listing 

yeah, but we should check if the capability is enabled (from the lister).

the other idea I had (other than creating the controller/informer dynamically) would be to terminate the operator when the capability was enabled at runtime - but that would be more complicated.

please document why we have decided to use the dynamic client.